### PR TITLE
Add database of bug bounty regex

### PR DESCRIPTION
**Pull request by bee-san**
Why?

More regex = better tooling. This PR adds a bunch specifically for bug bounties.

### CODE REVIEW

**Comment by bee-san** [comment]
Tests not added:

Things I couldn't find tests for:
* Facebook (no idea how to use their API)
* Twitter (I am bannned from making a bot)
* GitHub App Token does not exist according to google 
* LinkedIn Client ID - changed to hash
* LinkedIn Secret Key - changed to hash
* GitHub Access Token - same as personal access token?
* Stripe Restricted API Token - cant find
* Square Access Token - no examples
* Square OAuth Token - no examples
* PayPal/Braintree Access Token - no examples
* MWS Auth Token - no examples
* Picatic API Key - no examples
* Google OAuth Access Key - no example
* Google OAuth ID - no examples
* StackHawk API Key 
* NuGet API Key
* SendGrid Token
* Zoho Webhook Token
* New Relic Admin API Key
* New Relic Insights API Key
* New Relic REST API Key
* New Relic Synthetics Location Key
* Microsoft Teams Webhook
* Google FCM Server Key
* couldnt get google cal to work:

```
def test_google_cal():
    res = r.check(["https://calendar.google.com/calendar/embed?src=ht3jlfaac5lfd6263ulfh4tql8%40group.calendar.google.com&ctz=Europe%2FLondon"])
    _assert_match_in_items("Google Calendar URI", res)
```

* Discord Webhook
* Cloudinary Credentials
* Bitly secret key
* PyPi Upload Token
* Shopify Private App Access Token
* Shopify Custom App Access Token
* Shopify Access Token
* Shopify Shared Secret
* Dynatrace Token


* aws sns doesnt work:

```
def test_aws_sns():
    res = r.check(["arn:aws:sns:us-east-2:123456789012:MyTopic"])
    _assert_match_in_items("Amazon SNS Topic", res)
```

* Google Cloud Platform API Key
* Mailchimp API Key

File: README.md [hunk 1]
```diff
@@ -70,24 +69,84 @@ Say you have a `.pcap` file from a network attack. `What` can identify this and
...
+1. Download all GitHub repositories of an organisation
+2. Search for anything that you can submit as a bounty, like API keys
+
+```shell
+# Download all repositories
+GHUSER=CHANGEME; curl "https://api.github.com/users/$GHUSER/repos?per_page=1000" | grep -o 'git@[^"]*' | xargs -L1 git clone
+
+# Will print when it finds things.
+# Loops over all files in current directory.
+for f in $(find . ); do pywhat --include "Bug Bounty" $f; done
```
Comment (bee-san) [inline README.md:101]: ToDo: Check this works

File: README.md [hunk 2]
```diff
@@ -70,24 +69,84 @@ Say you have a `.pcap` file from a network attack. `What` can identify this and
...
+
+### 🕷 Scan all web pages for bounties
+
+```shell
+# Recursively download all web pages of a site
+wget -r -np -k https://skerritt.blog
+
+# Will print when it finds things.
+# Loops over all files in current directory.
+for f in $(find . ); do pywhat --include "Bug Bounty" $f; done
```
Comment (bee-san) [inline README.md:112]: ToDo: Check this works

File: tests/test_regex_identifier.py [hunk 3]
```diff
@@ -10,6 +10,21 @@
...
+    assert "Name" in r.distribution.get_regexes()[0]
+
+def _assert_match_in_items(name, res):
+    for i in res:
+        if i["Regex Pattern"]["Name"] == name:
+            assert True
+        else:
+            assert False
+
+@pytest.mark.skip(reason="Not all regex have tests now, check my PR for info.")
```
Comment (piatrashkakanstantinass) [inline tests/test_regex_identifier.py:27]: Which PR? A link would be helpful😅

File: README.md [hunk 4]
```diff
@@ -70,24 +69,84 @@ Say you have a `.pcap` file from a network attack. `What` can identify this and
...
+```
+
+**PS**: We support more filters than just bug bounties! Run `pywhat
```
Comment (piatrashkakanstantinass) [inline README.md:115]: `pywhat --help`

**Review by piatrashkakanstantinass — CHANGES_REQUESTED** [review]

File: tests/test_regex_identifier.py (hunk 3 above)
Comment (bee-san) [reply tests/test_regex_identifier.py]: This PR!

**Pull request merged**